composer.json

```diff
@@ -11,36 +11,39 @@
     }
   },
   "require": {
-    "php": "^7.0",
+    "php": "^7.1.3",
     "ext-PDO": "*",
     "ext-json": "*",
     "ext-sockets": "*",
-    "composer/composer": "@stable",
+    "composer/composer": "^1.4",
     "composer/semver": "^1.4",
     "graylog2/gelf-php": "^1.4.2",
     "guzzlehttp/guzzle": "^6.2",
-    "illuminate/config": "^5.5",
-    "illuminate/container": "^5.5",
-    "illuminate/contracts": "^5.5",
+    "illuminate/config": "^5.6",
+    "illuminate/container": "^5.6",
+    "illuminate/contracts": "^5.6",
     "magento/magento-cloud-components": "^1.0.1",
-    "magento/magento-cloud-docker": "^1.0",
     "monolog/monolog": "^1.16",
     "psr/container": "^1.0",
     "psr/log": "^1.0",
-    "symfony/console": "^2.6||^4.0",
-    "symfony/process": "~2.1||~4.1.0",
-    "symfony/yaml": "^3.3||^4.0",
-    "twig/twig": "^1.0||^2.0"
+    "symfony/console": "^2.3||^4.0",
+    "symfony/process": "^2.1||^4.1",
+    "symfony/serializer": "^4.2",
+    "symfony/yaml": "^4.0",
+    "twig/twig": "^2.0"
+  },
+  "suggest": {
+    "magento/magento-cloud-docker": "^1.0"
```

Comment:
Why not `require`? You will need to rewrite a good chunk of Docs to make it work

Reply:
It will be changed before release. Since Docker is not published ECE-Tools will fail locally without modification composer.json